mergify_cli/__init__.py

```diff
@@ -192,17 +206,45 @@ async def get_changeid_and_pull(
     return changeid, pull
 
 
-Change = typing.NewType("Change", tuple[ChangeId, str, str, str])
+@dataclasses.dataclass
+class Change:
+    id: ChangeId
+    pull: github_types.PullRequest | None
+
+
+@dataclasses.dataclass
+class LocalChange(Change):
+    commit_sha: str
+    title: str
+    message: str
+
+    @property
+    def pull_for_log(self) -> github_types.PullRequest:
+        if self.pull:
+            return self.pull
+        return PULL_MANNEQUIN_CREATION
```

Comment:
Any reason this is not on the `Change` class since it's already duplicated?

Reply:
OrphanChange and LocalChange don't have the same pull MANNEQUIN for logging.

All of this is temporary to refactor the code. At the end of my stack, these helpers vanish because the logging occurs only at one place, not four places like now.